dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1606,18 +1606,23 @@ a.download-video {
   }
 }
 
+.level-group-number {
+  display: block;
+  font-size: 16px;
+  line-height: 24px;
+  float: left;
+  width: 30px;
+}
+
+.level-group-content {
+  overflow: hidden;
```

Comment:
You can comment out this line to remove indentation and match the existing layout.